package.json

```diff
@@ -89,6 +89,7 @@
     "@salesforce/ts-types": "^1.5.20",
     "chalk": "^4.1.0",
     "csv-parse": "^4.16.3",
+    "csv-stringify": "^6.0.5",
```

Comment:
jsyk, we need to get approval for all 3rd party libs.  This one looks like it shouldn't be a problem.